src/components/settings/DataManagement/useGlobalImportFileParser.ts

```diff
@@ -10,10 +10,12 @@ import type { SettingsState } from '@/store/settingsSlice'
 import type { UndeployedSafesState } from '@/features/counterfactual/store/undeployedSafesSlice'
 
 import { useMemo } from 'react'
+import type { VisitedSafesState } from '@/store/visitedSafesSlice'
 
 export const enum SAFE_EXPORT_VERSION {
   V1 = '1.0',
   V2 = '2.0',
+  V3 = '3.0',
```

Comment:
What are the implications of this? Will V2 and V1 exports still be importable?

Reply:
V1 and V2 imports still work. It just imports whatever subset of properties is defined in the json.